ansible/roles/java-base/tasks/main.yml

```diff
@@ -34,11 +34,11 @@
 # if this fails you want to check in vars/main.yml and add package name
 # as appropriate -- try to use generic os family if available.
 - name: install java
-  when: not os|startswith("zos")
+  when: not os|startswith("zos") and "ppc64" not in ansible_nodename
   package: name="{{ java_package_name }}" state=present
 
 - name: install webupd8 oracle java 8 extras
-  when: os in ("ubuntu1204", "ubuntu1404")
+  when: os in ("ubuntu1204", "ubuntu1404") and "ppc64" not in ansible_nodename
```

Comment:
I think @jbarz has a PR (or at least one in progress) that does the right thing for ppc in respect to installing java8